src/libraries/System.IO.FileSystem.DriveInfo/tests/DriveInfo.Unix.Tests.cs

```diff
@@ -60,7 +60,7 @@ public void PropertiesOfInvalidDrive()
         }
 
         [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsNotWindowsSubsystemForLinux))] // https://github.com/dotnet/runtime/issues/18487
-        [PlatformSpecific(TestPlatforms.AnyUnix)]
+        [PlatformSpecific(TestPlatforms.AnyUnix & ~TestPlatforms.Android)]
```

Comment:
I think instead of disabling the test we should instead get the info from a partition we know exists on Android, e.g. `/data`

Reply:
The partition layout appears to be device specific, and not the same between simulator and device

Reply:
`/data` should always exist on Android 4.0+ though, but I'm fine to revisit this in a follow-up PR so we can get this one finally in :)